lintconfig.json

```diff
@@ -0,0 +1,97 @@
+{
+    "concurrency": 4,
+    "deadline": "600s",
+    "enable": [
+        "aligncheck",
+        "deadcode",
+        "errcheck",
+        "gas",
+        "goconst",
+        "gofmt",
+        "goimports",
+        "golint",
+        "ineffassign",
+        "interfacer",
+        "lll",
+        "megacheck",
+        "misspell",
+        "structcheck",
+        "unconvert",
+        "varcheck",
+        "vet",
+        "vetshadow"
+    ],
+    "enableGC": true,
+    "exclude": [
+        "vendor",
+        ".pb.go",
+        "mixer/pkg/config/proto/combined.go",
+        ".*.gen.go",
+        "should have a package comment",
+        ".*pkg/config/apiserver_test.go:.* method WriteHeaderAndJson should be WriteHeaderAndJSON",
+        "genfiles",
+        "broker/pkg/model/config/mock_store.go",
+        "broker/pkg/platform/kube/crd/types.go",
+        "broker/pkg/testing/mock/proto/fake_config.pb.go",
```

Comment:
why do we have to list all those .pb.go when there is line 27 already ?

Reply:
these things are auto collected by a script to cover autogen files, and therefore more comprehensive and some files which are not covered in the patterns (like `mock_store.go` etc above).

`.pb.go` pattern still exists, since there are actually pb.go files which are checked in but not generated through bazel. I think this situation should be fixed later.